app/actions/index.js

```diff
@@ -2,11 +2,19 @@ import _ from 'lodash'
 import { logDebug, logError } from '../util/logger'
 import client from '../client'
 
-export const addOracle = ({ address }) => dispatch => {
-  return client.addOracle({ address }).then(txHash => {
-    logDebug(`client.addOracle: tx:`, txHash)
+export const addDataFeed = ({ address }) => dispatch => {
+  return client.addDataFeed({ address }).then(txHash => {
+    logDebug(`client.addDataFeed: tx:`, txHash)
```

Comment:
do these only log if the app is running in a particular debug mode

Reply:
yep, we have this in futarchy-app as well